### ISSUE TITLE
Bless not working and creating empty source file

### ISSUE THREAD

**Issue by vinu4git** [BlessCSS/grunt-bless#14]
I have configured bless as follows

bless: {
            css: {
                options: {
                    compress: true,
                    force: false,
                    cacheBuster: true
                },
                files: {
                    '../consumer/theme/app.css': '../consumer/theme/out.css'
                }
            }
        },

Have added a watch on the css file to trigger bless. But the split does not happen the source file turns empty after tasks run from watch.

Blessc of the file actually works. Any idea what could be wrong?

![image](https://cloud.githubusercontent.com/assets/1969133/4003771/dacab6c0-2979-11e4-9903-1ef49241a68e.png)

**Comment by kerryj89** [comment]
That was my issue as well.  I was thinking the left side was the source and right was the destination.  I guess I should've read into the "input" directory name more in your example :P

**Comment by AaronAsAChimp** [comment]
Yes, thats why I leave this ticket open :D Grunt is the one handles the source/dest mappings so its out of my control. In the latest version (0.2.0), grunt-bless will refuse to overwrite the source file to prevent mistakes like this from destroying anything.

There are other, less confusing, ways of specifying source/dest mappings that you can read about [here](http://gruntjs.com/configuring-tasks#files).
